ArduPlane/altitude.cpp

```diff
@@ -163,13 +163,28 @@ float Plane::relative_ground_altitude(bool use_rangefinder_if_available, bool us
     return relative_altitude;
 }
 
+/*
+  return true if we should use the rangefinder for a specific use case
+ */
+bool Plane::rangefinder_use(enum RangeFinderUse use_rangefinder) const
+{
+    const uint8_t use = uint8_t(g.rangefinder_landing.get());
+    if (use == uint8_t(RangeFinderUse::NONE)) {
+        return false;
+    }
+    if (use == uint8_t(RangeFinderUse::ALL)) {
```

Comment:
Should this just check the first bit, rather than exact equality?

Reply:
yes, it should! fixed